packages/cli/commands/project/open.js

```diff
@@ -0,0 +1,70 @@
+const open = require('open');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+  addTestingOptions,
+} = require('../../lib/commonOpts');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  getProjectConfig,
+  getProjectDetailUrl,
+  verifyProjectExists,
+} = require('../../lib/projects');
+const { projectNamePrompt } = require('../../lib/prompts/projectNamePrompt');
+
+const i18nKey = 'cli.commands.project.subcommands.open';
+
+exports.command = 'open';
+exports.describe = i18n(`${i18nKey}.describe`);
+
+exports.handler = async options => {
+  await loadAndValidateOptions(options);
+
+  const accountId = getAccountId(options);
+  const { project } = options;
+  const { projectConfig } = await getProjectConfig();
+  let projectName = project;
+  if (projectName) {
+    const projectExists = await verifyProjectExists(accountId, projectName);
+    if (!projectExists) {
+      return;
+    }
+  } else if (!projectName && projectConfig) {
+    projectName = projectConfig.name;
+  } else if (!projectName && !projectConfig) {
```

Comment:
If no project option is provided and no project config exists, prompt the user for a name. Verify that a project exists with the name and default account, or the provided account using the `--account` option.